Lib/test/test_urllib.py

```diff
@@ -1526,6 +1526,24 @@ def test_quoting(self):
                          "url2pathname() failed; %s != %s" %
                          (expect, result))
 
+    @unittest.skipUnless(sys.platform == 'win32',
+                         'test specific to the nturl2path functions.')
+    def test_prefixes(self):
+        # Test special prefixes are correctly handled in pathname2url()
+        given = '\\\\?\\C:\\dir'
+        expect = '///C:/dir'
+        result = urllib.request.pathname2url(given)
+        self.assertEqual(expect, result,
+                         "pathname2url() failed; %s != %s" %
+                         (expect, result))
+        given = '\\\\?\\unc\\server\\share\\dir'
+        expect = '/server/share/dir'
```

Comment:
`pathname2url()` normally returns a URL with four leading slashes when given a standard UNC path. This is slightly nonstandard - nowadays `file:` URIs for UNC paths generally start with two slashes. But here the expected result has only a single leading slash - is that right @zooba? Sorry to ping about a 3y/o change!

Reply:
I think that's right. The format is `file://<server>/<share>/<path>` and you can omit the server for local machine, hence the common `file:///C/Local/Files` format.

The only other option would be to have no leading slash, so that it concatenates with `file://` directly. But I'm not 100% sure how this is meant to be used.

Reply:
The intention is to allow concatenation onto `file:` IIUC, hence `pathname2url(r'c:\foo')` correctly gives `'///c:/foo'`. I've logged a bug about this (and how UNC paths are handled more generally) and I'll request your review on the PR!